docs/datagrid/general.md

```diff
@@ -1,7 +1,8 @@
 DataGrid
 ===
 
-The DataGrid is a versatile tool for visualizing and editing tabular data.
+The DataGrid is a versatile tool registered together with the Highcharts Dashboards package for visualizing and editing tabular data.
+It can be a part of a dashboard or used as a standalone component.  
```

Comment:
Change to "It can be a part of a dashboard or used as a standalone component, but requires a Dashboards license."